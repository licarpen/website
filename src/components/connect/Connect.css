.Connect{
  margin: auto;
  margin-top: 5vw;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  width: 350px;
  background: rgb(68, 67, 67);
  padding: 18px;
  border-radius: 0.2em;

  a{
    font-size: 20px;
    font-family: Arial, Helvetica, sans-serif;
    margin-top: 15px;
    margin-bottom: 15px;
    color: white;
    text-decoration: none;
  }

  img{
    width: 60px;
    margin-right: 20px;
    margin-left: 20px;
  }

  @media screen and (max-width: 450px){
    width: 80%;
    img{
      width: 30px;
      margin-right: 2vw;
      margin-left: 1.5vw;
    }
    a{
      font-size: 18px;
    }
  }
}