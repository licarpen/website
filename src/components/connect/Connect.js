import React from 'react';
import styles from './Connect.css';
import github from '../../assets/connect/github.svg';
import linkedin from '../../assets/connect/linkedin.svg';
import email from '../../assets/connect/email.svg';

const Connect = () => (
  <div className={styles.Connect}>
    <a href='https://github.com/licarpen' target='_blank' rel='noopener noreferrer' ><img src={github}></img>/licarpen</a>
    <a href='https://www.linkedin.com/in/lisacarpenter256/' target='_blank' rel='noopener noreferrer'><img src={linkedin}></img>/lisacarpenter256</a>
    <a className={styles.email} href='mailto:carpenter.lisa@gmail.com' target='_blank' rel='noopener noreferrer'><img src={email}></img>carpenter.lisa@gmail.com</a>
  </div>
);

export default Connect;
