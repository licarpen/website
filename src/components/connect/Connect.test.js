import React from 'react';
import { shallow } from 'enzyme';
import Connect from './Connect';

describe('Connect component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Connect />);
    expect(wrapper).toMatchSnapshot();
  });
});
