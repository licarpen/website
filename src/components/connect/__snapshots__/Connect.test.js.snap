// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connect component matches a snapshot 1`] = `
<div
  className="Connect"
>
  <a
    href="https://github.com/licarpen"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      src="/path/image"
    />
    /licarpen
  </a>
  <a
    href="https://www.linkedin.com/in/lisacarpenter256/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      src="/path/image"
    />
    /lisacarpenter256
  </a>
  <a
    className="email"
    href="mailto:carpenter.lisa@gmail.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      src="/path/image"
    />
    carpenter.lisa@gmail.com
  </a>
</div>
`;
